core/AppRouter.js

```diff
@@ -146,10 +146,22 @@ export class AppRouter {
     AppRouter.notify();
     return routeData;
   }
+
+  static initPopstateListener() {
```

Comment:
Maybe call this automatically inside `createRouterData` or somewhere that will be exactly called by the user on init